src/impls.rs

```diff
@@ -104,7 +104,8 @@ where
 	T: Metadata + 'static,
 {
 	fn type_info() -> Type {
-		TypeVariant::new("Option", Namespace::prelude())
+		TypeVariant::new()
+			.path(Path::new().ident("Option"))
```

Comment:
I'd rather go for a special constructor so that users have to opt-in to create a Voldemord type which should be a rather rare thing instead of making users opt-in to create a common named type.

Reply:
E.g. we could have 4 constructors
1. `Path::voldemord`
2. `Path::prelude` (with prelude namespace)
3. `Path::new(name, namespace)` (or so)

Reply:
And maybe `Path::prelude` should actually be hidden from users.